packages/sdk-socket-server/redis-connection.js

```diff
@@ -0,0 +1,17 @@
+const Redis = require('ioredis');
+
+const redisClient = new Redis({
+  host: 'redis',
```

Comment:
'redis' hostname shouldn't be hardcoded. Same with port. We should have them in the .env file otherwise there willl be issue when deploying to cluster

Reply:
makes sense. thanks